packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -793,7 +801,7 @@ export class GarbageCollector implements IGarbageCollector {
 
         // If we are running in GC test mode, delete objects for unused routes. This enables testing scenarios
         // involving access to deleted data.
-        if (this.testMode) {
+        if (testMode) {
```

Comment:
Are you planning to use the `testMode` option for the inactive object used alert as well?
This code here will delete the inactive data stores (without wating for inactive timeout). So, this won't work for the alert.

Reply:
No, this is just to maintain support for GC Spy's button